tests/conftest.py

```diff
@@ -22,14 +26,18 @@ def anyio_backend():
 
 @pytest.fixture(scope="session")
 def local_grand_challenge() -> Generator[str, None, None]:
-    local_api_url = "https://gc.localhost/api/v1/"
+    local_api_url = os.environ.get(
+        "LOCAL_API_URL",
+        "https://gc.localhost/api/v1/",
```

Comment:
I use http://gc.localhost:8000 locally.

Reply:
Might be good to namespace the env var, `GCAPI_TESTS_LOCAL_API_URL` maybe?